app/Http/Controllers/MessagesController.php

```diff
@@ -2,20 +2,37 @@
 
 namespace Gladiator\Http\Controllers;
 
+use Gladiator\Models\Contest;
 use Gladiator\Models\Message;
 use Gladiator\Events\QueueMessageRequest;
 
 class MessagesController extends Controller
 {
     /**
+     * Display the specified resource.
+     *
+     * @param  string  $id  Contest ID
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        $messages = Message::where('contest_id', '=', $id)->get();
+
+        return view('messages.show', compact('messages'));
+    }
+
+    /**
      * Fire an event that queues a message to be sent.
      *
      * @param int $id
      */
-    public static function sendMessage($id)
+    public static function sendMessage(Message $message)
```

Comment:
Oh interesting. I was trying something similar and it wouldn't do that for me :( It's otherwise something that can be specified in the `RouteServiceProvider` I believe...